R/utils.R

```diff
@@ -0,0 +1,23 @@
+
+check_color <- function(clr) {
+  if (!is.null(clr) && !all(inherits(clr, c("crayon", "character")))) {
+    stop("All colors must be of class character or crayon",
+         call. = FALSE)
+  }
+  
+  # Change "rainbow" into roygbiv
+  clr <- multicolor::insert_rainbow(clr)
```

Comment:
this lline comes before the check4pkg fxn call, but it should come after it, correct?

Reply:
oh woops, very true!